deploy/chaos_crds.yaml

```diff
@@ -605,6 +605,7 @@ spec:
                           allowEmptyValue: false
                           minLength: 1
                   securityContext:
+                    x-kubernetes-preserve-unknown-fields: true
```

Comment:
can we remove the circle ci build operation looks like we are not using it and it's failing unnecessarily

Reply:
Hi, @chandankumar4 facing some issues to remove the checks, will remove for upcoming PRs. Thanks!